README.md

```diff
@@ -36,33 +35,31 @@ docker run \
   codeclimate/codeclimate help
 ```
 
-## Packages
-
-The above is very transparent. It's clear what's happening, and any changes
-required to work with your specific Docker setup can be discovered easily. That
-said, it can be unwieldy to invoke such a command on a regular basis.
+## Anywhere
 
-For this reason, we also provide packages that include a small wrapper script
-for the above invocation:
+If you are not using OS X:
 
-### OS X
-
-```console
-brew tap codeclimate/formulae
-brew install codeclimate
+* Run the following command:
 ```
-
-### Anywhere
-
-```console
 curl -L https://github.com/codeclimate/codeclimate/archive/v0.0.7.tar.gz | tar xvz
 cd codeclimate-* && sudo make install
 ```
+* Run `docker pull codeclimate/codeclimate`. Before continuing on, run `docker version` to verify that Docker is succesfully set up and running.
```

Comment:
`docker pull` and `docker run` are only needed if you don't use our wrapper package. If you run the `curl; make install` instructions you have pulled the image and `codeclimate` the command is available, now need to `docker pull` or `docker run`.

The basic categories would be this:

OS X suggested: run `brew ...` and use `codeclimate [cmd]`

Linux suggested: run `curl; make install` and use `codeclimate [cmd]`

Anywhere: run `docker pull` and use `docker run`.